Patient/PatientList.cs

```diff
@@ -12,9 +13,86 @@ namespace Doctor_Appointment_Management_System.Patient
 {
     public partial class PatientList : Form
     {
+        private SqlConnection databaseConnection;
         public PatientList()
         {
             InitializeComponent();
+            this.databaseConnection = Databse.DatabaseConnection.getConnection();
+        }
+
+        private void loadPatients()
+        {
+
+            // open database connection
+            Databse.DatabaseConnection.open();
+
+            // creare Datatable object
+            DataTable patientListDataTable = new DataTable();
+            SqlDataAdapter auserListSqlAdapter = new SqlDataAdapter("select patient_id as ID, first_name as [First Name], last_name as [Last Name], dob as DOB, address as Address, phone_number as [Phone Number] from Patient;", this.databaseConnection);
+
+            auserListSqlAdapter.Fill(patientListDataTable);
+            tblpatientgrid.DataSource = patientListDataTable;
+
+            //close database connection
+            Databse.DatabaseConnection.close();
+
+        }
+
+        private void btnadd_Click(object sender, EventArgs e)
+        {
+            PatientForm patientForm = new PatientForm();
+            patientForm.Show();
+        }
+
+        private void tblPatientList_CellContentClick(object sender, DataGridViewCellEventArgs e)
+        {
+
+        }
+
+        private void btnrefresh_Click(object sender, EventArgs e)
+        {
+            loadPatients();
+        }
+
+        private void btndelete_Click(object sender, EventArgs e)
+        {
+            DialogResult confirmResut = MessageBox.Show("Are you sure you want to delete this user?", "Delete User", MessageBoxButtons.YesNo);
+
+            int patientIdToDelete = 8;
+            if (confirmResut == DialogResult.Yes)
+            {
+                try
+                {
+                    SqlCommand patientInsertCommand;
+
+                    patientInsertCommand = new SqlCommand("DELETE [user] WHERE id=@id", this.databaseConnection);
+                    Databse.DatabaseConnection.open(); // open databse
+
+                    // bind values to delete query
+                    patientInsertCommand.Parameters.AddWithValue("@id", patientIdToDelete);
```

Comment:
`patientIdToDelete` varible shpold be declared after the line 16. 
Refer `UserList` form code on how to do this in development branch.

1. Checkout to development branch
2. Pull developmnt branch 
3. Now checkout to your branch
4. Right click on development branch and click merge development in to your branch

Now refer the `UserList` form code.